src/components/modal/Previewer.jsx

```
@@ -0,0 +1,39 @@
+import React from "react";
```

ESLint

[error] 'React' is defined but never used.

(no-unused-vars)